web/src/admin/policies/geoip/GeoIPPolicyForm.ts

```diff
@@ -0,0 +1,170 @@
+import { BasePolicyForm } from "@goauthentik/admin/policies/BasePolicyForm";
+import { DEFAULT_CONFIG } from "@goauthentik/common/api/config";
+import { first } from "@goauthentik/common/utils";
+import "@goauthentik/elements/forms/FormGroup";
+import "@goauthentik/elements/forms/HorizontalFormElement";
+import "@goauthentik/elements/forms/SearchSelect";
+
+import { msg } from "@lit/localize";
+import { TemplateResult, html } from "lit";
+import { customElement } from "lit/decorators.js";
+import { ifDefined } from "lit/directives/if-defined.js";
+
+import { App, GeoIPPolicy, PoliciesApi, TypeCreate } from "@goauthentik/api";
+
+@customElement("ak-policy-geoip-form")
+export class GeoIPPolicyForm extends BasePolicyForm<GeoIPPolicy> {
+    loadInstance(pk: string): Promise<GeoIPPolicy> {
+        return new PoliciesApi(DEFAULT_CONFIG).policiesGeoipRetrieve({
+            policyUuid: pk,
+        });
+    }
+
+    async send(data: GeoIPPolicy): Promise<GeoIPPolicy> {
+        if (data.asnMode?.toString() === "") {
```

Comment:
This is somewhat of a downside of the generated API client, but this value here should never be undefined so this if shouldn't be required

Reply:
Combined with the check below (which is still needed because `"".split(",")` is non-empty)